tools/src/test/resources/QualificationExpectations/db_sim_test_expectation.csv

```diff
@@ -1,2 +1,2 @@
-App Name,App ID,SQL DF Duration,SQL Dataframe Task Duration,App Duration,GPU Opportunity,Executor CPU Time Percent,SQL Ids with Failures,Unsupported Read File Formats and Types,Unsupported Write Data Format,Complex Types,Nested Complex Types,Potential Problems,Longest SQL Duration,NONSQL Task Duration Plus Overhead,Unsupported Task Duration,Supported SQL DF Task Duration,Task Speedup Factor,App Duration Estimated
-Spark shell,local-1623876083964,119903,1417661,133857,93094,91.14,"","","","","","",119903,9399,316964,1100697,2.92,false
+App Name,App ID,Recommendation,Estimated GPU Speedup,Estimated GPU Duration,Estimated GPU Time Saved,SQL DF Duration,SQL Dataframe Task Duration,App Duration,GPU Opportunity,Executor CPU Time Percent,SQL Ids with Failures,Unsupported Read File Formats and Types,Unsupported Write Data Format,Complex Types,Nested Complex Types,Potential Problems,Longest SQL Duration,NONSQL Task Duration Plus Overhead,Unsupported Task Duration,Supported SQL DF Task Duration,Task Speedup Factor,App Duration Estimated
+Spark shell,local-1623876083964,Recommended,1.8426441236879585,72643.97844337518,61213.02155662482,119903,1417661,133857,93094,91.14,"","","","","","",119903,9399,316964,1100697,2.92,false
```

Comment:
these should only have 2 decimal places...

Reply:
Done!